internal/resources/policy/test_helper.go

```diff
@@ -82,7 +82,7 @@ func NewScopeHelperResources() *ScopeHelperResources {
 			ResourceVar:           clusterResourceVar,
 			ResourceName:          fmt.Sprintf("%s.%s", clusterResource, clusterResourceVar),
 			KubeConfigPath:        os.Getenv("KUBECONFIG"),
-			Name:                  clusterName,
+			Name:                  acctest.RandomWithPrefix(clusterNamePrefix),
```

Comment:
The clean-up of the resources is taken care by the framework right ?

Reply:
Yes, I checked for the cluster used, it is cleaned up.